src/traces/sankey/attributes.js

```diff
@@ -122,7 +121,14 @@ var attrs = module.exports = overrideAll({
             role: 'style',
             description: 'Sets the thickness (in px) of the `nodes`.'
         },
-        hoverinfo: extendFlat({}, plotAttrs.hoverinfo, {flags: []}),
+        hoverinfo: {
+            valtype: 'enumerated',
+            values: ['all', 'none', 'skip'],
+            dflt: 'all',
+            role: 'info',
+            editType: 'none',
```

Comment:
`overrideAll` is just going to wipe this out... if you really want to set `editType` per attribute, just stop using `overrideAll`, and just set it explicitly in each attribute.

Reply:
Do we even need `overrideAll` anymore?

Reply:
probably not, just make sure that if you take it out, `restyle` works correctly with attributes you set to an `editType` less than `calc`.